projects/angular/src/data/datagrid/datagrid.ts

```diff
@@ -484,4 +464,47 @@ export class ClrDatagrid<T = any> implements AfterContentInit, AfterViewInit, On
   dataChanged() {
     this.items.refresh();
   }
+
+  private setupVirtualScroll() {
```

Comment:
I think this would fit better as we are toggling on and of subscriptions.
```suggestion
  private toggleVirtualScrollSubscriptions() {
```

Reply:
Corrected in commit 83b19e2